mavros_extras/src/plugins/visualization.cpp

```diff
@@ -91,7 +93,15 @@ class VisualizationPlugin : public MavRosPlugin {
 
 		tf::Transform transform;
 		transform.setOrigin(tf::Vector3(pos_ned.y, pos_ned.x, -pos_ned.z));
-		transform.setRotation(uas->get_attitude_orientation());
+		geometry_msgs::QuaternionStamped q_body;
+		tf::quaternionTFToMsg(uas->get_attitude_orientation(), q_body.quaternion);
+		q_body.header.frame_id = child_frame_id;
+		q_body.header.stamp = uas->synchronise_stamp(pos_ned.time_boot_ms);
+		geometry_msgs::QuaternionStamped q_inertial;

```

Comment:
Minor change, make all the "inertial" to "ground" as that's what it says in ros spec. People might be confused.